packages/core/src/components/Input/Input.tsx

```diff
@@ -20,28 +19,32 @@ const Input = ({
   validationSchema,
   status = 'basic',
   disabled = false,
-  append,
+  startElement,
+  endElement,
 }: InputProps) => {
   if (!register) {
     throw new Error('Input component must be used within a Form component');
   }
 
-  const showStatusIcon = () => !disabled && !append;
+  const showStatusIcon = () => !disabled && !endElement;
 
   return (
     <InputGroup css={[mapColors[status]]} disabled={disabled}>
+      {startElement ? <div css={S.startElement}>{startElement}</div> : null}
       <InputBase
         type={type}
         id={`formElement-${name}`}
         placeholder={placeholder}
         disabled={disabled}
+        $paddingLeft={startElement && 40}
```

Comment:
We don't have a dollar notation (`$propName`) for props in the project. So please remove the dollar sign for these two props. 

And my suggestion here is to just use the `css` prop. Something like this:
```
css={{
  paddingLeft: startElement && 40,
  paddingRight: endElement && 40
}}
```

This allows to remove `getPadding()` function: 
just set `padding-left` and `padding-right` in the `InputBase`.